### PR TITLE
Add schema for ACL: APP_ACL_TABLE and APP_ACL_RULE_TABLE (#325)

### PR DESCRIPTION
**Pull request by shine4chen**
Signed-off-by: shine.chen <shine.chen@nephosinc.com>

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -39,6 +39,8 @@ namespace swss {
 #define APP_VXLAN_VRF_TABLE_NAME        "VXLAN_VRF_TABLE"
 #define APP_VXLAN_TUNNEL_MAP_TABLE_NAME "VXLAN_TUNNEL_MAP_TABLE"
 #define APP_VXLAN_TUNNEL_TABLE_NAME     "VXLAN_TUNNEL_TABLE"
+#define APP_ACL_TABLE_NAME              "APP_ACL_TABLE"
+#define APP_ACL_RULE_TABLE_NAME         "APP_ACL_RULE"
```
Comment (prsunny) [inline common/schema.h]: APP_ACL_RULE_TABLE

Comment (stcheng) [reply common/schema.h]: do we want the APP_ prefix?

**Comment by prsunny** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

File: common/schema.h (hunk 1 above)
Comment (stcheng) [reply common/schema.h]: change to
```
#define APP_ACL_TABLE_TABLE_NAME        "ACL_TABLE_TABLE"
#define APP_ACL_RULE_TABLE_NAME         "ACL_RULE_TABLE"
```

**Comment by shine4chen** [comment]
@stcheng  Sure , will correct it soon.

**Comment by shine4chen** [comment]
> `APP_ACL_TABLE_NAME` shall be changed to `APP_ACL_TABLE_TABLE_NAME`
> the first table is for ACL_TABLE the second table is for TABLE_NAME

**Comment by shine4chen** [comment]
> `APP_ACL_TABLE_NAME` shall be changed to `APP_ACL_TABLE_TABLE_NAME`
> the first table is for ACL_TABLE the second table is for TABLE_NAME
@stcheng  If we follow the "CFG_ACL_TABLE_NAME" name criteria APP_ACL_TABLE_NAME is suitable.

**Comment by stcheng** [comment]
I see. I think CFG_ACL_TABLE_NAME is not correct. I'll discuss internally and get back.

**Comment by lguohan** [comment]
@stcheng, do you still have concern?

**Comment by stcheng** [comment]
@shine4chen I have fixed the `CFG_ACL_TABLE_NAME` issue, could you update the `APP_ACL_TABLE_NAME` as well?

**Comment by stcheng** [comment]
by the way, could you add the link to the design for adding these two tables?

**Comment by shine4chen** [comment]
@stcheng  This is the [Appdb-acl-table](https://github.com/Azure/SONiC/blob/master/doc/acl/ACL-High-Level-Design.md#312-app-db) definition.  I will change `APP_ACL_TABLE_NAME` to `APP_ACL_TABLE_TABLE_NAME` and resolve the confilcts.

**Comment by stcheng** [comment]
may i ask which application is writing to these two tables? what's the different purposes for having both configuration ACL table and application ACL table?

**Comment by shine4chen** [comment]
Currently MCLAG apps will use these two acl table to dynamically install/uninstall  acl rule for traffic isolation.  Generally the static configured acl rules user defines will be put on cfg table. The dynamic acl rules apps defines will be put on app table.  @stcheng

**Comment by stcheng** [comment]
Thanks for the response!

**Pull request merged**